### PR TITLE
Quick fix for issue #28

### PR DESCRIPTION
**Pull request by sn8to**
Have not tested the code, but hopefully (fingers crossed may or may not be needed) will work just fine. Fixes an issue of which opening and closing KShare repeatedly on Windows clients will create separate processes.

### CODE REVIEW

**Comment by sn8to** [comment]
Unofficial 100% success on the checks, only reason why AppVeyor is saying no is that it's throwing a hissy fit about building a Windows program in Qt and not VS

**Comment by ArsenArsen** [comment]
Make it `com.arsenarsen.kshare` and it's a merge

**Comment by sn8to** [comment]
Fixed. I also almost forgot the includes for QLockFile, that would've been a bummer.
https://github.com/GlitchMasta47/KShare/commit/fd7a670a7ac0d6ef692fcd5634602de9cfb0ca7f

File: main.cpp [hunk 1]
```diff
@@ -66,6 +68,11 @@ int main(int argc, char *argv[]) {
     a.setApplicationName("KShare");
     a.setOrganizationName("ArsenArsen");
     a.setApplicationVersion("4.1");
+    
+    QLockFile lockFile(QDir::temp().absoluteFilePath("arsenarsen.kshare.lock"));
```
Comment (ArsenArsen) [inline main.cpp]: Doesn't look different to me :p

Comment (sn8to) [reply main.cpp]: See my comment, I can't believe I forgot it

**Comment by sn8to** [comment]
https://github.com/GlitchMasta47/KShare/commit/cdd65002ac0b2986e30c241a12929e8784fca55b

**Comment by ArsenArsen** [comment]
LGTM

**Comment by ArsenArsen** [comment]
Just noticed the base was master and not dev. That made some conflicts. Can you rebase onto dev?

**Comment by sn8to** [comment]
#49

**Pull request closed without merging**